docs/src/examples/components/Animation/Types/AnimationExample.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+import { Animation, Icon } from '@stardust-ui/react'
+
+const AnimationExample = () => (
+  <Animation name="spinner">
```

Comment:
All the `Animation` examples are theme dependent. Would it make more sense to add `Provider` to the examples and explicitly define the animations there?
Not sure about that, would like to hear other opinions.

Reply:
I understand your point, but I don't think this is good idea, because we will have much code in the examples, for something that is documented in the Theming guide. How the animation are created will be document as part of the theming guidem and the default theme will have few animations that will be shown in the examples. Let's hear the opinion from the other folks too. @kuzhelov @Bugaa92 @alinais @levithomason @layershifter

Reply:
I have a similar problem in #451, we definitely need to show theme specific stuff because examples can be broken when you will try to switch to another theme.

> I understand your point, but I don't think this is good idea, because we will have much code in the examples, for something that is documented in the Theming guide. 

This is a valid point, too.

Reply:
After offline discussion we decided to add the Provider with the theme. All examples as well as the Theming.tsx guide page, now contains a Provider where the animations are defined.